### ISSUE TITLE
Config file examples: irr.net mirror list

### ISSUE THREAD

**Issue by pforemski** [irrdnet/irrd#380]
Hi,

Pardon if that's silly for any reason, but could mirroring all IRRs listed at http://www.irr.net/docs/list.html be made a bit easier for first-time users? I thought that maybe an illustrative YAML file somewhere in the repo, with all these sources configured, could do. Just an idea.

Thanks for your great work!

**Comment by mxsasha** [comment]
Yes, I think this (or some form of it) is a good idea. Better config examples is one of the things on my list for a documentation improvement run.

**Comment by job** [comment]
This may sound strange, but I am not sure whether such an example config file helps.

People need to make a conscious choice which databases to mirror, the listing at `irr.net` contains a bunch of databases which are poorly maintained. Also, some database operators require you to a-priori tell them which source IP addresses to add to an allowlist before mirroring can work.

**Comment by mxsasha** [comment]
I wasn't thinking of making a config exactly for everything listed on irr.net, but mainly more guidance or starting point for a few different scenarios. This has also become more complex as the configuration options grow.
